backend/src/resolvers/comments.js

```diff
@@ -0,0 +1,52 @@
+import { neo4jgraphql } from 'neo4j-graphql-js'
+import { UserInputError } from 'apollo-server'
+
+const COMMENT_MIN_LENGTH = 1
+export default {
+  Query: {
+    CommentByPost: async (object, params, context, resolveInfo) => {
+      const { postId } = params
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-17T09:39:57Z" title="Wednesday, April 17th 2019, 11:39:57 am +02:00">Apr 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,12 @@
+import { neo4jgraphql } from 'neo4j-graphql-js'
+
+export default {
+  Mutation: {
+    CreateComment: (object, params, context, resolveInfo) => {
+      return neo4jgraphql(object, params, context, resolveInfo, false)
+    },
+    AddPostComments: (object, params, context, resolveInfo) => {
```


Please do not create a Comment without a Post. The frontend should assign the Post of the Comment already during creation. We don't want any left-over state, ie. comments without authors or posts.
```suggestion
```

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-04-17T10:39:43Z" title="Wednesday, April 17th 2019, 12:39:43 pm +02:00">Apr 17, 2019</time>_

---


I do not want to leave it like this @roschaefer, which is why I said :point_up: I am not happy with it, but I spent a couple hours last night and could not seem to get it working, and there are no examples of creating any nodes passing in additional information.

We have a few where we assign an author, but that does not require passing in any additional info as we can access the user id with the context. 

When I try to modify the resolver, I run into the issues stated :point_up: